mobile/lib/main.dart

```diff
@@ -60,15 +63,22 @@ void main() {
 
   FLog.applyConfigurations(config);
   runApp(MultiProvider(providers: [
-    ChangeNotifierProvider(create: (context) => TradeValuesChangeNotifier(TradeValuesService())),
-    ChangeNotifierProvider(create: (context) => AmountDenominationChangeNotifier()),
-    ChangeNotifierProvider(create: (context) => SubmitOrderChangeNotifier(OrderService())),
-    ChangeNotifierProvider(create: (context) => OrderChangeNotifier(OrderService())),
     ChangeNotifierProvider(
-        create: (context) => PositionChangeNotifier(PositionService(), OrderService())),
-    ChangeNotifierProvider(create: (context) => WalletChangeNotifier(const WalletService())),
+        create: (context) => TradeValuesChangeNotifier(TradeValuesService())),
     ChangeNotifierProvider(
-        create: (context) => CandlestickChangeNotifier(const CandlestickService())),
+        create: (context) => AmountDenominationChangeNotifier()),
```

Comment:
❓ It's odd that there are so many format changes. Do you use different formatting settings?

Reply:
I'll try to reformat it tomorrow, maybe Android Studio didn't pick that one up.